entries/1/1stdomains.nz.json

```diff
@@ -0,0 +1,15 @@
+{
+  "1st Domains": {
+    "domain": "1stdomains.nz",
+    "img": "1stdomains.nz.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.1stdomains.co.nz/afmviewfaq.asp?faqid=95",
+    "keywords": [
+      "domains",
+      "hosting",
+      "email"
+    ]
```

Comment:
```suggestion
    ],
    "regions": [
      "nz"
    ]
```

Reply:
Thanks @Carlgo11 

While we are based in New Zealand we have customers worldwide, would adding the region restrict how we are viewed in the directory?